recipes/setuptools/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "22.0.5" %}
```

Comment:
LOL they just bumped to 23.0.0. I wonder if we will ever release the current version. 😆